README.md

```diff
@@ -84,35 +84,35 @@ defined('YII_ENV') or define('YII_ENV', 'dev');
 // Alternatives set this in your rr.yaml file
 //defined('YII_ENV') or define('YII_ENV', \getenv('YII_ENV'));
 
-// Load your vendor dependencies and Yii2 autoloader
-require_once '/path/to/vendor/autoload.php';
-require_once '/path/to/vendor/yiisoft/yii2/Yii.php';
+require __DIR__ . '/../vendor/autoload.php';
+require __DIR__ . '/../vendor/yiisoft/yii2/Yii.php';
 
-// Roadrunner relay and PSR7 object
-$relay = new \Spiral\Goridge\StreamRelay(STDIN, STDOUT);
-$psr7 = new \Spiral\RoadRunner\PSR7Client(new \Spiral\RoadRunner\Worker($relay));
+$worker = Spiral\RoadRunner\Worker::create();
+$psrFactory = new Nyholm\Psr7\Factory\Psr17Factory();
+$psr7 = new Spiral\RoadRunner\Http\PSR7Worker($worker, $psrFactory, $psrFactory, $psrFactory);
 
-// Load your configuration file
-$config = require_once '/path/to/config/config.php';
+$config = require __DIR__ . '/../config/web.php';
 
 $application = (new \yii\Psr7\web\Application($config));
 
 // Handle each request in a loop
-while ($request = $psr7->acceptRequest()) {
+while (true) {
```

Comment:
I think  introducing a separate PSR17 handler as an example is unnecessary - for this library I'm not really wanting to promote a separate library.

Something like this for the main loop would probably be sufficient?

```php
while ($request = $psr7->waitRequest()) {
    if (($request instanceof Psr\Http\Message\ServerRequestInterface)) {
        try {
              $response = $application->handle($request);
              $psr7->respond($response);
        } catch (\Throwable $e) {
               // Handle an invalid PSR7 response here. Logging, or just returning the error directly.
              $psr7->getWorker()->error((string)$e);
        }

        if ($application->clean()) {
            $psr7->worker()->stop();
            return;
        }
    }
}
```

Reply:
Although I prefer to use `if` for early leaving the loop but, sure, I will adapt the way of coding you use